src/test/java/seedu/address/logic/commands/visit/AddVisitCommandTest.java

```diff
@@ -107,6 +113,83 @@ public void execute_duplicateVisit_throwsCommandException() {
                 addvisitCommand.execute(modelStub));
     }
 
+    @Test
+    public void execute_infectedVisit_successWithWarning() {
+        ModelStubAcceptingVisitAdded modelStub =
+                new AddVisitCommandTest.ModelStubAcceptingVisitAdded();
+        Visit visitWithInfected = new VisitBuilder().withPerson(INFECTED_PERSON).build();
+        try {
+            CommandResult commandResult = new AddVisitCommand(INFECTED_PERSON.getId(), DEFAULT_LOCATION_INDEX,
+                    DEFAULT_DATE).execute(modelStub);
+
+            assertEquals(String.format(AddVisitCommand.MESSAGE_INFECTED_MADE_VISIT, visitWithInfected),
+                    commandResult.getFeedbackToUser());
+            assertEquals(Arrays.asList(visitWithInfected), modelStub.visitsAdded);
+        } catch (CommandException e) {
+            assert false : "Command Exception not expected.";
```

Comment:
Suggest replacing with an existing method inside command utils that handles commands. The method name is AssertCommandSuccess()

Reply:
Done